Unix/installbuilder/conf/omiserver.conf

```diff
@@ -60,6 +60,6 @@ configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv3=false
 
 ## non-root feature parameters
-nonroot=true
-service=omiservice
-restartEngine=true
+#nonroot=true
+#service=omi
+#restartEngine=true
```

Comment:
I think having the logic for this flag is wrong, and we should rely on the service controller to recover (i.e. the root server component should just exit if the child exits for some reason unexpectedly). The service control managers already have "backoff logic" to handle this well, where OMI code has no such logic (nor should it).

Additionally, we need to understand how quickly the selector loop in the child (non-prived) server will run. We shouldn't allow the service control managers to try to restart so quickly that it would likely fail anyway.